scripts/p2p-test-tool.py

```diff
@@ -414,7 +414,7 @@ def test_repl_load(container):
             cmd = (f"sudo docker run -u root --rm -it -v {repl_node[i]['volume'].name}:{args.rnode_directory} "
                    f"--cpuset-cpus={args.cpuset_cpus} --memory={args.memory} --name {repl_node[i]['name']} "
                    f"--network {args.network} {args.image} "
-                   f"--grpc-host {container.name} -r")
+                   f"--grpc-host {container.name} repl")
```

Comment:
Isn't it --repl ?

Reply:
Did we change the API?

Reply:
they are changing to "express" command or whatever. This is like cisco or other advanced consoles that autocomplete

Reply:
https://github.com/rchain/rchain/pull/801

Reply:
eval, repl and deploy demo changed. You also have to add run when starting because it is broken up.